terraform/datadog/etcd.tf

```diff
@@ -0,0 +1,23 @@
+resource "datadog_monitor" "etcd_one_leader" {
+  name                = "${format("%s etcd IsLeader count is exactly one (%s)", var.env, element(list("upper", "lower"), count.index))}"
+  type                = "query alert"
+  message             = "There is not exactly one etcd reporting IsLeader."
+  escalation_message  = "There is still not exactly one etcd reporting IsLeader."
+  notify_no_data      = false
+  require_full_window = true
+
+  count = 2
+
+  query = "${
+    format(
+      "min(last_5m):sum:cf.etcd.IsLeader{deployment:%s}.fill(last, 60) %s",
+      var.env,
+      element(list("> 1", "< 1"), count.index)
+    )}"
+
+  thresholds {
+    critical = "${element(list("1", "1"), count.index)}"
```

Comment:
I also found it quite confusing to read. Can you change it to a single value please?